src/libraries/System.Private.CoreLib/src/System/Globalization/CultureInfo.Windows.cs

```diff
@@ -0,0 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Globalization
+{
+    public partial class CultureInfo : IFormatProvider
+    {
+        internal static CultureInfo GetUserDefaultCulture() => NlsGetUserDefaultCulture();
```

Comment:
Do we need this extra layer of forwarding - would it be better to just have the implementation that calls the right Windows APIs directly here?

Reply:
> Do we need this extra layer of forwarding - would it be better to just have the implementation that calls the right Windows APIs directly here?

You're right, we don't need it. Thanks.

Reply:
Done. Thanks for catching this